PandaGrom/hometask-5/4.5.sql

```diff
@@ -0,0 +1,2 @@
+SELECT name, COUNT(language.language_id) FROM film 
```

Comment:
hen you write a query with joins it's better to specify attributes of which tables you are selecting
like `language.name` not just `name`